usr/share/device-quirks/scripts/quirk-identification-triage.sh

```diff
@@ -0,0 +1,39 @@
+#/bin/bash
+### SYSTEM IDENTIFICATION SCRIPT
+
+SYS_ID="$(cat /sys/devices/virtual/dmi/id/product_name)"
+VENDOR="$(cat /sys/devices/virtual/dmi/id/board_vendor)"
+
+### DEVICE LISTS
+# OXP Devices
+OXP_LIST="ONE XPLAYER:ONEXPLAYER 1 T08:ONEXPLAYER 1S A08:ONEXPLAYER 1S T08:ONEXPLAYER mini A07:ONEXPLAYER mini GA72:ONEXPLAYER mini GT72:ONEXPLAYER GUNDAM GA72:ONEXPLAYER 2 ARP23"
+
+# AYANEO AIR Devices
+AYANEO_LIST="AIR:AIR Pro"
+
+# AOKZOE Devices
+AOK_LIST="AOKZOE"
+
+### CALL DEVICE SPECIFIC SCRIPTS
+# OXP Devices
+if [[ ":$OXP_LIST:" =~ ":$SYS_ID:"  ]]; then
+    echo "Possible OneXPlayer Device FOUND"
+    ./oxp/oxp_table.sh
```

Comment:
please add a dummy script for now or comment these out so we have a good target for other devices but it wont crash.

Reply:
Dummy files added